apps/demo/demo.py

```diff
@@ -60,7 +61,7 @@ def __init__(self, filename) -> None:
         t = ["scroll demo. press escape to quit", "arrow keys move"]
 
         # save the rendered text
-        self.text_overlay = [f.render(i, 1, (180, 180, 0)) for i in t]
+        self.text_overlay = [f.render(i, True, (180, 180, 0)) for i in t]
```

Comment:
2nd position = antialias, I guess the 1 is equal to True, right?

Reply:
yep